src/server/src/test/resources/cassandra-reaper-postgres-at.yaml

```diff
@@ -9,6 +9,7 @@ hangingRepairTimeoutMins: 1
 storageType: postgres
 incrementalRepair: false
 jmxConnectionTimeoutInSeconds: 300
+datacenterAvailability: LOCAL
```

Comment:
any reason to not use the default of `ALL` ?
(i would think it's only the cassandra backend that we want to test with `LOCAL`)